web/pom.xml

```diff
@@ -50,9 +50,9 @@
          <artifactId>javax.servlet-api</artifactId>
       </dependency>
       <dependency>
-         <groupId>javax.servlet</groupId>
-         <artifactId>jsp-api</artifactId>
-      </dependency>
+	    <groupId>javax.servlet.jsp</groupId>
+	    <artifactId>javax.servlet.jsp-api</artifactId>
```

Comment:
seeing some undesired white spaces

Reply:
Please @mozzy11 may you point to the lines that have white space?